download/releaseNotes/releaseNotes7.0.adoc

```diff
@@ -195,3 +195,82 @@ This example also demonstrates real-time producing and consuming of tasks.
 * `HardMediumSoftBigDecimalScore`: 3 score levels of BigDecimal scores. Contributed by Brad Hards.
 * The public API is now a solid foundation for the 7.x years.
+
+== Workbench
+
+=== Enhanced solver editor
+
+OptaPlanner Solver editor screen now supports adding all termination types, including composite termination.
+Phase configuration section allows you to tweak Construction Heuristic settings and select Local Search alogorithm
+to optimize your planning problem.
+
+image:7.0/solverEditorScreen.png[Solver editor screen]
+
+=== Guided rule editor integration
+
+Remember the pain of defining a constraint match in rules created by Guided rule editor? The process required creating
+a global variable `scoreHolder` and using `Free form line` input to insert Java code to modify the score.
+
+[source,java,options="nowrap"]
+----
+scoreHolder.addSoftConstraintMatch(kcontext, -$computer.cost);
+----
+
+The procedure is a lot simpler now. A correct list of all available score operations is provided in the action list of
+a guided rule, based on the selected score type in your planning solution object. Just define the value of a constraint.
+
+image:7.0/guidedRule.png[Guided rule editor]
+
+=== Difficulty comparator definition
+
+OptaPlanner domain editor now provides a way to specify how difficult is to assign a planning entity.
+Several Construction Heuristic algorithms require this information in order to construct a good initial solution.
+
+image:7.0/difficultyComparator.png[Difficulty comparator]
+
+=== Improved example import dialog
+
+Default examples are now part of the Workbench war file and fetching them from external location is no longer needed (but still supported).
+Leverage tag-based filtering to quickly access projects from the field you are interested in.
+
+image:7.0/exampleImport.png[Example import dialog]
+
+=== Other
+* OptaPlanner Workbench now fully supports bendable score types
+* It is no longer required to define `plannermgmt` role for the Workbench user to access OptaPlanner UI items.
+Use role-based permission management screen instead to control the access.
+
+image:7.0/permissionScreen.png[Permission screen]
+
+* Two new examples have been added. A course scheduling example (curriculumcourse) models lecture scheduling at universities.
+Employee rostering example (employee-rostering) optimizes task assigning to employees based on their skill.
+
+== Execution server
+
+=== Simplified REST API
+
+`ServiceResponse` wrapper has been removed from OptaPlanner service responses returned by KIE Execution Server.
+This allows easier processing of the responses on the client side.
+
+Get best solution response before:
+
+[source,xml,options="nowrap"]
+----
+<org.kie.api.model.ServiceResponse>
+    <type>SUCCESS</type>
+    <msg>Best computed solution...</msg>
+    <result class="solver-instance">
+    ...
+    <best-solution class="curriculumcourse.curriculumcourse.CourseSchedule">
+    </best-solution>
```

Comment:
Are you sure this is correct?
IIRC, the `best-solution` element was (and still is) nested inside the `result` element.